code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -90,6 +90,8 @@
 			if(stream_mode)
 				if(isliving(T))
 					var/mob/living/M = T
+					if(!M.can_inject())
+						return

```

Comment:
this should prob be `continue` instead

Reply:
continue means continue to the next item in the loop, not continue to the next block of code below

Reply:
ok i guess